projects/packages/connection/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.41.8] - 2022-07-29
+
```

Comment:
```suggestion
- Updated package dependencies

```

Reply:
Ah yes - the release process did warn me there was one with no content! Good catch.